src/test/fuzz/script.cpp

```diff
@@ -108,12 +108,7 @@ FUZZ_TARGET_INIT(script, initialize_script)
     (void)ScriptToAsmStr(script, false);
     (void)ScriptToAsmStr(script, true);
```

Comment:
unrelated, but could also use ConsumeBool here

Reply:
This has been done in this commit fae3f178238df96554dc2495e040f5580b55408a

Marking as resolved.